test/jasmine/tests/modebar_test.js

```diff
@@ -1406,6 +1445,50 @@ describe('ModeBar', function() {
             });
         });
 
+        describe('map handlers', function() {
+            it('@gl button *resetViewMapbox* should reset the map view attribute to their default', function(done) {
+                var gd = createGraphDiv();
+
+                function _assert(centerLon, centerLat, zoom) {
+                    var mapLayout = gd._fullLayout.map;
+
+                    expect(mapLayout.center.lon).toBe(centerLon, 'center.lon');
+                    expect(mapLayout.center.lat).toBe(centerLat, 'center.lat');
+                    expect(mapLayout.zoom).toBe(zoom, 'zoom');
+                }
+
+                Plotly.newPlot(gd, [{
+                    type: 'scattermap',
+                    lon: [10, 20, 30],
+                    lat: [10, 20, 30]
+                }], {
+                    map: {
+                        center: {lon: 10, lat: 10},
+                        zoom: 8
+                    }
+                }, {})
+                .then(function() {
+                    _assert(10, 10, 8);
+
+                    return Plotly.relayout(gd, {
+                        'map.zoom': 10,
+                        'map.center.lon': 30
+                    });
+                })
+                .then(function() {
+                    _assert(30, 10, 10);
+
+                    var button = selectButton(gd._fullLayout._modeBar, 'resetViewMapbox');
```

Comment:
@archmoj Is this supposed to say `Mapbox` here or should it be `Map`?

Reply:
Thanks. Now I see this other test :pray:
Addressed by [1dd8c0e](https://github.com/plotly/plotly.js/pull/7060/commits/1dd8c0e58c6ac87bef3d10198606b374bf951bbf).